tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "alwaysStrict": true,
+    "noUnusedLocals": true,
+    "experimentalDecorators": true,
+    "skipLibCheck": true,
+    "strictNullChecks": true,
+    "declaration": true,
+    "outDir": "dist/",
+    "lib": [
+      "es2015",
+      "es6",
+      "es2017.object",
+      "dom"
+    ],
+    "target": "es5",
```

Comment:
We can safely transpire it to version `es2017`. This way we will use native async/await instead of using ES5 workarounds.